modules/gles2/CMakeLists.txt

```diff
@@ -43,3 +43,7 @@ add_deqp_module(deqp-gles2 "${DEQP_GLES2_SRCS}" "${DEQP_GLES2_LIBS}" tes2TestPac
 # Data directories
 add_data_dir(deqp-gles2 ../../data/gles2/data		gles2/data)
 add_data_dir(deqp-gles2 ../../data/gles2/shaders	gles2/shaders)
+
+if (EMSCRIPTEN)
+	append_linker_flag(deqp-gles2 "--shell-file ${CMAKE_SOURCE_DIR}/emscripten/shell.html")
+endif()
```

Comment:
I think it would make sense to add this to add_deqp_module, if it's likely to be the same shell for each module.

Reply:
I was pondering on this exact question as well while writing this, and my thinking was that perhaps the shell might change per test module, e.g. different modules might have buttons for running different tests, so the shell file for each module might list the set of tests in that module, or something like that. That's why I initially added the shell file only for the GLES2 suite.